apiserver/worker/worker.py

```diff
@@ -62,6 +62,15 @@
 """
 
 
+UPLOAD_ERROR_MESSAGE = """
+We had some trouble uploading your bot; likely the final compiled bot
+was too large (> 100 MiB). If you cannot figure out why this happened,
+please email us at halite@halite.io. We can help.
+
+For our reference, here is the trace of the error:
+"""
+
```

Comment:
Given that the size is now explicitly checked before upload, I think I'd leave out the part about "likely the final compiled bot was too large".